src/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ add_library(teakra
     ahbm.h
     apbp.cpp
     apbp.h
+    bit_set.h
```

Comment:
The file name was like this in citra because its main purpose was for the class `BitSet`. As we don't have this class here, a better name would just name this `bit.h`